verilog/analysis/verilog_linter_configuration.h

```diff
@@ -130,6 +130,30 @@ struct ProjectPolicy {
   std::string ListPathGlobs() const;
 };
 
+struct LinterConfigurationFlags {
+  // strings, ints, bools, and unprocessed values from flags, no other derived
+  // information. Reasonable default values may be specified here for each
+  // member
+
+  // The base set of rules used by linter
+  const RuleSet& ruleset;
+  // Bundle of rules to enable/disable in addition to the base set
+  const RuleBundle& rules;
+  // Path to a file with extra linter configuration, applied on top of the
+  // base 'ruleset' and extra 'rules'
+  absl::string_view config_file;
```

Comment:
I would not store string_views for the config file, linting_start_file and waivers_file.
The struct is meant to be passed around with the configuration, but we don't know anything about the lifetime of the strings we set it to. So all these should be std::string.

Reply:
Right, changed to strings